test/langtools/tools/javac/diags/examples.not-yet.txt

```diff
@@ -214,4 +214,4 @@ compiler.err.preview.without.source.or.release
 compiler.misc.illegal.signature                               # the compiler can now detect more non-denotable types before class writing
 
 # this one needs a forged class file to be reproduced
-compiler.err.annotation.unrecognized.attribute.name
\ No newline at end of file
+compiler.err.annotation.unrecognized.attribute.name
```

Comment:
Is it just me? I cannot see any difference in these lines...  Is there a difference in trailing whitespace?

Reply:
IIRC, it was an end-of-file difference, but it isn't being reflected in the webrev either.